GothicVRProject/GothicVR
# Feature/286 optimize dialog system and idle animations

This branch includes some quality of life changes for NPCs, their animations, and Dialogs. Basically test the following:

# To test
1. Spawn Bloodwyn (Id=233) and set time to 8:58 - Is he walking his arrogant walk through the camp at 9:0?
2. Spawn Mud (Id=574) - Is he sitting on his bench and breathing heavily?
3. Walk into mud, execute one of his dialog options and see him follow you if you go away. (Hint: He will walk through the wall after a few seconds, that's ok for now)

# Hint:
* Do not build for MVR. It's currently broken. But this is handled in another fix branch by @JucanAndreiDaniel 
* But if you run from Editor, it works fine.

# Checklist

## Scene

# Testing
